Tests/Command/CleanCommandTest.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+/*
+ * This file is part of the FOSOAuthServerBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\OAuthServerBundle\Tests\Command;
+
+use FOS\OAuthServerBundle\Command\CleanCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class CleanCommandTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var \FOS\OAuthServerBundle\Command\CleanCommand
+     */
+    private $command;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|\Symfony\Component\DependencyInjection\ContainerInterface
+     */
+    private $container;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        $command = new CleanCommand();
+
+        $application = new Application();
+        $application->add($command);
+
+        $this->container = $this->getMock('Symfony\Component\DependencyInjection\ContainerInterface');
+
+        $this->command = $application->find($command->getName());
+        $this->command->setContainer($this->container);
+    }
+
+    /**
+     * Delete expired tokens for classes that implement the TokenManagerInterface.
+     */
+    public function testItShouldRemoveExpiredTokensForTokenManagerInterfaces()
+    {
+        $expiredAccessTokens = 5;
+        $accessTokenManager = $this->getMock('FOS\OAuthServerBundle\Model\TokenManagerInterface');
```

Comment:
I would prefer to use a ``@dataProvider`` to avoid duplication.
```php
/**
 * @dataProvider classProvider
 */
public function testMyFeature($class) {
    $this->getMock($class);
    // ...
}

public function classProvider()
{
    return array(
        array('FOS\OAuthServerBundle\Model\TokenManagerInterface'),
        array('FOS\OAuthServerBundle\Model\AuthCodeManagerInterface'),
    );
}
```